cli/requests/requests.go

```
@@ -0,0 +1,108 @@
+	return respOrError(&resp)
+}
+
+func RotateClaimToken() (*types.ApiResponse[types.UserResponse], error) {
+	resp := callDashboardApi[types.UserResponse]("POST", "/web/settings/rotate-claim-token")
+	return respOrError(&resp)
+}
+
+func createError[T any](resp *types.ApiResponse[T]) error {
+	if resp.Type != "success" {
+		if len(resp.Errors) > 0 {
+			return errors.New(fmt.Sprintf("API request failed. Message: %s", resp.Errors[0]))
```

golangci-lint (1.62.2)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)